### ISSUE TITLE
Add circular relationship

### ISSUE THREAD

**Issue by Moggla** [PeWu/topola-viewer#178]
Left side: two branches (currently)
Right side: circular relationship (this can also happen outside of Alabama)
![circular_relationship](https://github.com/user-attachments/assets/55df1e08-a57a-4703-8d0d-d702f128aedf)

**Comment by nyarlatothep** [comment]
Yes, happens in my tree view multiple times and makes that chart much more confusing.

Don't know how to fix this. Anybody out here  can help, because otherwise topola is such a beauty.

**Comment by Moggla** [comment]
I’m closing this issue because the underlying problem belongs to the topola repository, not topola‑viewer.
I have a (vibe) fix on my fork here:
https://github.com/Moggla/topola
